site/src/_pages/spec/1_block-types.mdx

```diff
@@ -266,30 +329,36 @@ Another special set of functions provided to blocks relate to managing links bet
-- MUST contain EITHER:
-  - `destinationEntityId` \[_string_] – the id of a single entity the link is made to, OR
-  - `aggregate` – an aggregation operation which the embedding application should resolve the link to, following the structure of the `operation` object [described above](#aggregate-entities-operation).
-- if `destinationEntityId` is defined:
-  - MAY contain `destinationEntityVersionId` to pin the link to a specific version of the destination entity.
-  - MAY contain `destinationEntityTypeId`: \[_string_]: the `entityTypeId` of the destination entity.
-- MAY contain `index` \[_integer_]: the position of this link in a list (for where ordering of links is important).
+- MUST contain:
+  - `sourceEntityId` \[_string_]: the `entityId` of the source entity.
+  - `path` \[_string_]: the path to the field on the source entity this link is conceptually made on, expressed as a [JSON path](https://goessner.net/articles/JsonPath/).
+  - EITHER:
+    - `destinationEntityId` \[_string_] – the id of a single entity the link is made to, OR
+    - `aggregate` – an aggregation operation which the embedding application SHOULD resolve the link to, following the
+- MAY contain
+
+  - `sourceEntityAccountId` \[_string_]: the `accountId` of the source entity.
+  - `sourceEntityTypeId`: \[_string_]: the `entityTypeId` of the source entity.
+  - `sourceEntityVersionId` \[_string_]: optionally specify that this link is only from a specific version of the source entity.
+
+- if `destinationEntityId` is defined, MAY contain:-
+  - `destinationEntityAccountId` \[_string_]: the `accountId` of the destination entity.
+  - `destinationEntityTypeId`: \[_string_]: the `entityTypeId` of the destination entity.
+  - `destinationEntityVersionId` to pin the link to a specific version of the destination entity.
+  - `index` \[_integer_]: the position of this link in a list (for where ordering of links is important).
 
 Once created, a `Link` includes a `linkId`.
 
-**E.g.1.** creating a `Link` with the following data indicates that this particular user should be linked to a company with id `company1`, and that the link conceptually is made on the user’s `employer` field – although when delivering data to blocks the resolved data for entity `company1` will be provided _alongside_ the user to the block, in the `linkedEntities` array, rather than injected into the properties of the user, and the link itself available in the `links` array provided to the block.
+**E.g.1.** creating a `Link` with the following data indicates that this particular user should be linked to a company with id `company1`, and that the link conceptually is made on the user’s `employer` field – although when delivering data to blocks the resolved data for entity `company1` will be provided _separately_ to `user1`, in the `linkedEntities` array, rather than injected into the properties of the user, and the link itself available in the `linkGroups` array provided to the block.
```

Comment:
I never quite understood "E.g.1" and "E.g.2" -- can't we just do:

```
For example:
1. Lorem ipsum
2. Lorem ipsum
```

Reply:
Done